airflow-core/src/airflow/ui/src/i18n/locales/nl/dashboard.json

```diff
@@ -0,0 +1,38 @@
+{
+  "welcome": "Welkom",
+  "stats": {
+    "stats": "Statistieken",
+    "failedDags": "Mislukte Dags",
+    "runningDags": "Lopende Dags",
+    "activeDags": "Actieve Dags",
+    "queuedDags": "Dags in de wachtrij"
+  },
+  "health": {
+    "health": "Gezondheid",
```

Comment:
I would make this "Status", rather than "Gezondheid". We don't use "Gezondheid" in this context.